### PR TITLE
Update product-technical-writing-team-handbook.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated dev reviewer process section.

### CODE REVIEW

File: operations/research-and-development/product/product-technical-writing-team-handbook.md [hunk 1]
```diff
@@ -92,13 +92,19 @@ Mattermost documentation covers a number of different topics. For documentation,
 
 **Dev Review**
 
-* Admin Guide - Enterprise: Scott Bishel \(@sbishel\)
-* Admin Guide - Non-enterprise: Joram Wilander \(@jwilander\)
-* End User Guide - Mobile and desktop app: Elias Nahum \(@enahum\)
-* End User Guide - Other: Dean Whillier \(@deanwhillier\)
+It's not always clear which developer should be added as a reviewer for a pull request. In some cases you may be prompted to add a reviewer based on them having recently viewed or edited the file you're submitting in the PR. If not, one of the team leads listed below can be added as a reviewer for documentation relevant to their area of expertise:
```
Comment (jasonblais) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md]: @justinegeffen What is the message/communication you intend to convey with this paragraph?

**Review by jasonblais — CHANGES_REQUESTED** [review]
One clarifying question, else looks good to me

File: operations/research-and-development/product/product-technical-writing-team-handbook.md (hunk 1 above)
Comment (justinegeffen) [reply operations/research-and-development/product/product-technical-writing-team-handbook.md]: I'm working on a reworded version now, good callout.

Comment (justinegeffen) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md]:
```suggestion
If your change requires dev review add the developer/s you've been working with as the reviewer/s. If you're unsure who to add as a dev reviewer you can select one of the team leads below: 
```

Comment (justinegeffen) [reply operations/research-and-development/product/product-technical-writing-team-handbook.md]: I've reworded it and it should be clearer.

**Review by jasonblais — APPROVED** [review]
Thanks!

**Pull request merged**